news/2018-05-02-spring-2018-fanart-contest-results.md

```diff
@@ -0,0 +1,89 @@
+
+I also feel compelled to note that the depiction of the osu!team cockpit relates to all of us on a deeply personal level. Excellent entry!
+
+[![](/wiki/shared/news/2018-05-02-spring-2018-fanart-contest-results/Heidy.jpg)](https://assets.ppy.sh/contests/64/winners/osu%21%20Spring%20Fanart%202018%20-%20Heidy.jpg)
+
+Scoring **5301** votes, [Heidy](https://osu.ppy.sh/users/7425199)'s beautifully illustrated take on Mocha and Pippi enjoying some sun and candy is serene, simple, and absolutely beautiful.
+
+[![](/wiki/shared/news/2018-05-02-spring-2018-fanart-contest-results/Evans.jpg)](https://assets.ppy.sh/contests/64/winners/osu%21%20Spring%20Fanart%202018%20-%20Evans.png)
+
+**5004** votes. [Evans](https://osu.ppy.sh/users/11967425). Mocha. Cute.
+
+It is just too adorable to describe any more than that. Check it out!
+
+[![](/wiki/shared/news/2018-05-02-spring-2018-fanart-contest-results/Vio.jpg)](https://assets.ppy.sh/contests/64/winners/osu%21%20Spring%20Fanart%202018%20-%20Vio.jpg)
+
+Sunset overtones herald **4771** votes for [Vio](https://osu.ppy.sh/users/5661237)'s wonderfully unique entry, showcasing Mocha enjoying a ride on her bike and appreciating the local wildlife. A beautifully realised setting that just *oozes* personality. Excellent work!
+
+[![](/wiki/shared/news/2018-05-02-spring-2018-fanart-contest-results/R3_AL.jpg)](https://assets.ppy.sh/contests/64/winners/osu%21%20Spring%20Fanart%202018%20-%20R3_AL.png)
+
+Achieving **3664** votes, [R3\_AL](https://osu.ppy.sh/users/11983357)'s adorable drawing of a rather aptly titled spring Pippi showcases her wonderfully bright-eyed gaze highlighted with vibrant colouring.
+
+[![](/wiki/shared/news/2018-05-02-spring-2018-fanart-contest-results/RaudhrAkai.jpg)](https://assets.ppy.sh/contests/64/winners/osu%21%20Spring%20Fanart%202018%20-%20RaudhrAkai.png)
+
+On wings of a spring-tinged dream, **3647** voters were enthralled by [RaudhrAkai](https://osu.ppy.sh/users/1631442)'s fantastical take on an otherwordly Pippi enjoying some springtime scenery. Well done!
+
+---------
+
+Like what you see? [Download all of the entries bundled together in high quality here!](https://assets.ppy.sh/contests/64/winners/osu%21%20Spring%20Fanart%202018%20Winners.zip)
+
+We'd like to thank everyone who entered for helping make the contest so great - this particular fanart contest was accompanied with a LOT of calls for more votes across the board, highlighting the number of incredibly talented entries we got to see.
+
+Each finalist snags themselves 2 months of osu!supporter for themselves or a friend, and the winning pieces are available **NOW** in-game as the Spring seasonal background pack, and will remain live for several weeks. Show the artist of your favourite piece some support and check out their profiles!
```

Comment:
I have no idea what the difference is between the winning pieces and finalists from this post

Reply:
made this more clear